src/errors.h

```diff
@@ -1778,8 +1778,8 @@ EXTERN char e_can_only_compare_list_with_list[]
 	INIT(= N_("E691: Can only compare List with List"));
 EXTERN char e_invalid_operation_for_list[]
 	INIT(= N_("E692: Invalid operation for List"));
-EXTERN char e_list_or_class_required_for_argument_nr[]
-	INIT(= N_("E693: List or Class required for argument %d"));
+EXTERN char e_class_or_typealias_required_for_argument_nr[]
+	INIT(= N_("E693: Class required for argument %d"));
```

Comment:
Oops; changing.

Reply:
Now I remember. I guess it's an implementation detail. But in the code, a class typealias is never seen (AFAICT). I guess it gets converted early so that all the normal class stuff works. Changed it to
```
E693: Class or class typealias required for argument %d
```
So it's clear when the code is
```
vim9script
class C
endclass
type T2 = list<string>
var v = instanceof(C.new(), T2)
```
```